src/components/OverflowMenu/OverflowMenu.js

```diff
@@ -166,17 +169,15 @@ export default class OverflowMenu extends Component {
             description={iconDescription}
             style={{ width: '100%' }}
           />
-          {!open ? null : floatingMenu ? (
+          {!containerWithChildrenWithProps || !floatingMenu ? (
```

Comment:
Is this block basically saying the following:

```
{!open &&
  !floatingMenu
    ? containerWithChildrenWithProps
    : <FloatMenu />
}
```

Reply:
@joshblack Oops I had a mistake translating the old logic to the new one. Fix pushed: https://github.com/carbon-design-system/carbon-components-react/pull/465/commits/9d2120deba9baf6c6423adbf2ce6360bd026a7ea

To address your original question - It’s rendering the menu body when it’s open, `null` otherwise. The menu body is wrapped as floating menu as necessary.